public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -43,6 +53,7 @@ export function PanelHeader({ panel, error, isViewing, isEditing, data, alertSta
         scopedVars={panel.scopedVars}
         links={getPanelLinksSupplier(panel)}
         error={error}
+        multipleErrors={multipleErrors}
```

Comment:
Nit: maybe a name like `hasMultipleErrors` would communicate better that this is a boolean prop

Reply:
I removed this property at the end, it's enough to have the `error`